docs/documentation/head/setup.md

```diff
@@ -42,3 +42,6 @@ Where MM is the major version, nn is the minor version and pp is the patch versi
 Versions for JDBC3 and lower can be found [here](https://jdbc.postgresql.org/download.html#others)
 This is a very brief outline of how to build the driver. Much more detailed information can be 
 found on the [github repo](https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md)
+
+Even though the JDBC driver should be built with Gradle, for situations, where use of Gradle is not possible, e.g., when building pgjdbc for distributions, the pgjdbc provides a convenience source release artifact **-src.tar.gz** - a Maven based project. The Maven based project contains a minimal version of the JDBC driver with reduced feature and test set and is still validly buildable within the Maven build environment.
```

Comment:
"the pgjdbc" is a little ambiguous. How about the "pgjdbc gradle build" ? or even "the gradle build" ?

Reply:
Corrected. And I have also split the text into more lines for easier diffs.